torch/_logging/_internal.py

```diff
@@ -148,11 +229,11 @@ def set_logs(
 
     log_state.clear()
 
-    if modules is None:
-        modules = {}
+    modules_ = {} if modules is None else modules
 
     def _set_logs(**kwargs):
-        for alias, val in itertools.chain(kwargs.items(), modules.items()):
+        for alias, val in itertools.chain(kwargs.items(), modules_.items()):
+            print(f"{alias}: {val}")
```

Comment:
These changes are not needed. Better to roll them back.

Reply:
Oops, didn't mean to keep that print. But the `modules_` change was to fix a mypy error due to assigning an empty dict to `modules`. Maybe there's a better way to fix that though

Reply:
Right. In that case it may be better to silence MyPy in that line, as I think it's cleaner to change the type of `modules` from optional to non-optional  as you did or via `modules = modules or {}` than having to create a new variable, which may be more error prone.